program-runtime/src/loaded_programs.rs

```diff
@@ -382,6 +382,19 @@ impl LoadedProgram {
     }
 
     pub fn to_unloaded(&self) -> Option<Self> {
+        #[cfg(test)]
```

Comment:
imo not great that we never test the real variants, we're testing the tests :P

Reply:
Yes `LoadedProgramType::TestLoaded` should be removed, but that is for another PR.